Model/Sales/Repository/PackageRepository.php

```diff
@@ -223,11 +280,11 @@ private function getAttributesFromProduct($tableName, $product){
         return $attributeValue;
     }
 
-	private function getAttributeId($connection, $tableName) {
+	private function getAttributeId($connection, $tableName, $databaseColumn) {
 		$sql = $connection
 			->select('entity_type_id')
 		    ->from($tableName)
-		    ->where('attribute_code = ?', 'myparcel_fit_in_mailbox');
+		    ->where('attribute_code = ?', 'myparcel_'. $databaseColumn);
```

Comment:
```suggestion
		    ->where('attribute_code = ?', 'myparcel_' . $databaseColumn);
```
Moet je even in phpstorm instellen

Reply:
Done!